als-server/jvm/src/main/scala/org/mulesoft/als/server/lsp4j/extension/SerializationParams.java

```diff
@@ -4,9 +4,19 @@
 
 public class SerializationParams {
     private TextDocumentIdentifier documentIdentifier;
+    private Boolean clean;
+    private Boolean sourcemaps;
+
+    public SerializationParams(TextDocumentIdentifier documentIdentifier, Boolean clean, Boolean sourcemaps) {
```

Comment:
Maybe we can use primitive types for avoid null assignments (boolean instead of Boolean)